proto/umee/incentive/v1/tx.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "umee/incentive/v1/incentive.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive/types";
+
+// Msg defines the x/incentive module's Msg service.
+service Msg {
+  // Claim defines a method for claiming any pending incentive rewards.
+  rpc Claim(MsgClaim) returns (MsgClaimResponse);
+
+  // Lock defines a method for locking uToken collateral into a locking tier.
+  rpc Lock(MsgLock) returns (MsgLockResponse);
+
+  // BeginUnbonding defines a method for starting to unbond uToken collateral.
+  rpc BeginUnbonding(MsgBeginUnbonding) returns (MsgBeginUnbondingResponse);
+
+  // Sponsor defines a permissionless method for sponsoring an incentive program.
+  rpc Sponsor(MsgSponsor) returns (MsgSponsorResponse);
+
+  // CreateProgram is used by governance proposals to create (but not fund) an incentive program.
+  rpc CreateProgram(MsgCreateProgram) returns (MsgCreateProgramResponse);
+
+  // CreateAndSponsorProgram is used by governance proposals to create and fund an incentive program.
+  rpc CreateAndSponsorProgram(MsgCreateAndSponsorProgram) returns (MsgCreateAndSponsorProgramResponse);
```

Comment:
These separate flows for creating incentive programs through governance give us multiple options for how to create external incentives when the time comes - either funding them on creation from a predetermined address (probably community fund) or creating the incentive program but letting any regular account use `MsgSponsor` to fund it after it passes.